bpy-build.yaml

```
@@ -0,0 +1,13 @@
+addon_folder: MCprep_addon
+build_name: MCprep_addon
+
+install_versions:
+  - '3.5'
+  - '3.4'
```

This may as well be a record of all the versions we intend to support. Though it should be important it prints no more than warnings if a user doesn't have a given version installed

```suggestion
  - '3.6'
  - '3.5'
  - '3.4'
  - '3.3'
  - '3.2'
  - '3.1'
  - '3.0'
  - '2.93'
  - '2.90'
  - '2.80'
```